frontend/src/metabase/parameters/components/ParameterSettings/ParameterSettings.unit.spec.tsx

```diff
@@ -97,6 +97,128 @@ describe("ParameterSidebar", () => {
 
     expect(onChangeQueryType).toHaveBeenCalledWith("none");
   });
+
+  describe("location", () => {
+    it("should render type", () => {
+      setup({
+        parameter: createMockUiParameter({
+          type: "string/=",
+          sectionId: "location",
+        }),
+      });
+
+      expect(screen.getByDisplayValue("Location")).toBeInTheDocument();
+    });
+
+    it("should render operator", () => {
+      setup({
+        parameter: createMockUiParameter({
+          type: "string/=",
+          sectionId: "location",
+        }),
+      });
+
+      expect(screen.getByDisplayValue("Is")).toBeInTheDocument();
+    });
+  });
+
+  describe("id", () => {
+    it("should render type", () => {
+      setup({
+        parameter: createMockUiParameter({
```

Comment:
We could save some LOC if parameter was created in the each `describe`, and then `it` would only have `setup({ parameter });` and the assertion.

Reply:
4618bdb9dbffe13a0f68d0502701f36b79bcc6ae